src/scripts.js

```diff
@@ -361,10 +363,19 @@ function addOrRemoveToPantry(user) {
         pantryTable.innerHTML += `
             <div class="boxI">${value.Ingredient}</div>
             <div class="boxA">${value.Amount}</div>
-            <button type="button" class="miniButtons">-</button>
-            <button type="button" class="miniButtons">+</button>
             `
     })
     return amount
 }
 
+function addItemToPantry() {
+    const foundIt = ingredients.ingredients.reduce((acc, element) => {
+        if(element.name === selectIngredient.value) {
+                acc = element.id
+        }
+            return acc
+        }, 0)
+        postItNote = {userID: user.id, ingredientID: foundIt, ingredientModification: inputQuantity.value}
```

Comment:
I console logged this and it came out just like the format, however according to the "Endpoints (GET and POST)" doc, I think ingredientModification is supposed to be a number not a string. We could do `Number(inputQuantity.value)` to change it to a number. Not important now, just a note.

Reply:
Good catch, Kiko!

Reply:
Good catch, just updated it